easy-ui-react/src/Button/utilities.ts

```diff
@@ -17,3 +18,16 @@ export function logWarningIfInvalidColorVariantCombination(
     );
   }
 }
+
+/**
+ * Removes aria-specific props from being applied to button DOM element.
+ */
+export function omitReactAriaSpecificProps(props: object) {
```

Comment:
i don't love this. let me know if you have any better ideas.

the problem is that `useButton(props)` is swallowing some props that need to get passed into the `<button />` element. specifically the `onPointerEnter` and `onPointerLeave` events that are being supplied by the Tooltip. they get passed into `useButton()` but don't come out the other side.

Reply:
i think what you did here is sensible